tests/wpunit/Tribe/Utils/Global_ID_Test.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+namespace Tribe\Utils;
+
+use \Tribe__Utils__Global_ID as Global_ID;
+
+/**
+ * Tests general Meetup functionality
+ */
+class Global_ID_Test extends \Codeception\TestCase\WPTestCase {
+
+	/**
+	 * @test
+	 *
+	 * @group utils
+	 */
+	public function it_should_be_instantiatable() {
+		$global_id = new Global_ID;
+
+		$this->assertInstanceOf( 'Tribe\Event_Aggregator\Utils\Global_ID', $global_id );
```

Comment:
If `$global_id` is a `Global_ID` (alias of pseudo-namespaced `Tribe__Utils__Global_ID`, how can its type be the properly namespaced `Tribe\Event_Aggregator\Utils\Global_ID`?